src/community/ogcapi/ogcapi-core/src/main/java/org/geoserver/api/APIFilterParser.java

```diff
@@ -17,35 +19,53 @@
 
     private static final FilterFactory2 FF = CommonFactoryFinder.getFilterFactory2();
     public static String CQL_TEXT = "cql-text";
+    public static String CQL_OBJECT = "cql-object";
 
     /**
-     * Parses the filter over the supported filter languages (right now, only {@link #CQL_TEXT}) and
-     * defaults the geometry liters in spatial filters to CRS84
+     * Parses the filter over the supported filter languages (right now, only {@link #CQL_TEXT} and
+     * {@link #CQL_OBJECT}) and defaults the geometry liters in spatial filters to CRS84
```

Comment:
Not part of the change, but liters doesn't make sense in this sentence. Maybe geometry reference?

Reply:
It's a typo, was supposed to be ``literals``.

Reply:
fixed